app/routers/quiz.py

```
@@ -0,0 +1,73 @@
+from app.services.quiz_service import create_quiz, get_quiz_questions
+
+router = APIRouter()
+logger = logging.getLogger(__name__)
+
+
+@router.post(
+    "/quizzes", response_model=QuizResponse, status_code=status.HTTP_201_CREATED
+)
+async def create_quiz_endpoint(
+    quiz_in: QuizCreateRequest,
+    db: AsyncSession = Depends(get_db),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)